services/QuillLMS/app/models/school.rb

```diff
@@ -129,7 +129,7 @@ def students
   end
 
   def self.school_year_start(time)
-    time.month >= 8 ? time.beginning_of_year + 7.months : time.beginning_of_year - 5.months
+    time.month >= 7 ? time.beginning_of_year + 6.months : time.beginning_of_year - 6.months
```

Comment:
Good catch, but let's use a named variable rather than the magic number `7` here, so that future developers better know your intent. While you're at it, how about replacing `6` with `HALF_A_YEAR = 6`?